sync_crawler/crawlers/base_crawler.py

```diff
@@ -20,3 +24,10 @@ def read(self, start_from: datetime) -> Iterable[News]:
             Iterable of News.
         """
         pass
+
+    def hash(self, data: str) -> str:
+        return hashlib.sha1(data.encode("utf-8")).hexdigest()
```

Comment:
**Suggestion:** Replace the direct use of `hashlib.sha1` with a more secure hash function such as `hashlib.sha256` to enhance security. [security, importance: 9]
```suggestion
        return hashlib.sha256(data.encode("utf-8")).hexdigest()
```

Reply:
It is just used to check for duplication, so we don't worry about security issues.